RSSReader-Maruyama/Serviceis/AuthenticationService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  AuthenticationService.swift
+//  RSSReader-Maruyama
+//
+//  Created by 丸山翔 on 2020/10/02.
+//  Copyright © 2020 丸山翔. All rights reserved.
+//
+
+import UIKit
+
+enum AuthenticationType: String, Codable {
+    case apple
+    case facebook
+    case google
+}
+
+enum SocialServiceAuthError: Error {
```

Comment:
```suggestion
enum AuthenticationServiceAuthError: Error {
```
AuthenticationServiceのサービスなのにSocialServiceAuthErrorは変な気がします。

Reply:
修正しました。